test/integration-tests/tests/warnings.test.ts

```diff
@@ -0,0 +1,90 @@
+import { PolarisExtensions, PolarisWarning } from '@enigmatis/polaris-common';
+import { PolarisServer } from '../../../src';
+import { initializeDatabase } from '../test-server/data-initalizer';
+import { graphqlRawRequest } from '../test-server/graphql-client';
+import { startTestServer, stopTestServer } from '../test-server/test-server';
+import * as simpleQuery from './jsonRequestsAndHeaders/simpleQuery.json';
+
+let polarisServer: PolarisServer;
+
+describe('warnings tests - shouldAddWarningsToExtensions is true', () => {
+    beforeEach(async () => {
+        const customWarnings: PolarisWarning[] = ['warning 1', 'warning 2'];
```

Comment:
This should not be in the before.
The way you should test this logic, is to activate a resolver that adds warning, and check that the extensions in the response contains warnings.